pylib/spinnaker/transform_old_config.py

```diff
@@ -20,6 +20,13 @@
 
 
 class Processor(object):
+  def __collect_environ(self, content):
+    result = {}
+    for match in re.finditer('^([^\s]+?)=([^\n]*)(?:\#.+)?', content,
+                             re.MULTILINE):
+      result[match.group(1)] = match.group(2)
```

Comment:
You could do some pythonic dictionary construction like `result = {match.group(1): match.group(2) for match in re.finditer(...)}`. If there are no matches this dictionary comprehension would still return `{}`.